docs/primer.md

```diff
@@ -13,3 +15,314 @@
+
+### Different Force of Exposure and Natural Protection
+
+If both assumptions discussed above are broken at once, such that the force of exposure differs by vaccination status and the focal pathogen provides natural protection, then 1 -- OR gives:
+
+$$VE = 1 - \frac{C_{\text{FV}}C_{\text{NU}}}{C_{\text{FU}}C_{\text{NV}}} = 1 - \frac{\varepsilon_{\text{NU}}}{\varepsilon_{\text{NV}}}\left\lbrack \left( 1 - \varphi \right)\frac{1 - e^{- \varepsilon_{\text{FV}}\lambda_{\text{FU}}t}}{1 - e^{- \varepsilon_{\text{FU}}\lambda_{\text{FU}}t}} + \varphi\theta_{\pi}\frac{1 - e^{- \varepsilon_{\text{FV}}\lambda_{\text{FV}}t}}{1 - e^{- \varepsilon_{\text{FU}}\lambda_{\text{FU}}t}} \right\rbrack \neq \varphi\left( 1 - \theta \right)$$
+
+In this case, the bias in the TND estimate of VE depends on the relative force of exposure between vaccinated and unvaccinated people, for all pathogens. Unlike in Section 6.1, assuming that vaccination correlates with force of exposure identically for different pathogens ($\frac{\varepsilon_{\text{FV}}}{\varepsilon_{\text{FU}}} = \frac{\varepsilon_{\text{NV}}}{\varepsilon_{\text{NU}}}$) no longer corrects this.
+
+Two further observations about these assumptions are also warranted:
+
+- The disparity in force of exposure by vaccination status is what biases 1 -- OR. Different forces of exposure between the focal and non-focal pathogens (i.e. $\varepsilon_{F} \neq \varepsilon_{N}$), even if these forces are dynamic in time (i.e. $\varepsilon_{F}\left( t \right) \neq \varepsilon_{N}(t)$), do not contribute to the bias.
+
+- The disparity in natural protection between the focal and non-focal pathogens need not be so exaggerated (i.e. complete vs. non-existent, respectively). Any disparity in natural protection will bias 1 -- OR for the same reasons described in Section 6.2, albeit perhaps to a lesser extent.
+
+### Behavioral Correlations between Vaccination and Susceptibility/Severity
+
+Even when $1 - OR = \varphi\left( 1 - \theta \right) = \varphi\left( 1 - \frac{\lambda_{\text{FV}}}{\lambda_{\text{FU}}}\frac{\pi_{\text{FV}}}{\pi_{\text{FU}}} \right)$, bias may be lurking in the interpretation. VE is typically interpreted as purely a result of vaccine-stimulated immune defense. But perhaps $\lambda_{\text{FV}} \neq \lambda_{\text{FU}}$ and/or $\pi_{\text{FV}} \neq \pi_{\text{FU}}$ for reasons unrelated to immunological action. For example:
+
+- Individuals at high risk of infection given exposure (e.g. elderly, some immunocompromised) may be more likely to get vaccinated than the general public. In this case, the realized value of $\theta_{\lambda} = \frac{\lambda_{\text{FV}}}{\lambda_{\text{FU}}}$ is closer to 1 than it would be if it only captured immunological phenomena.
```

Comment:
```suggestion
- Individuals at high risk of infection given exposure (e.g. elderly, some immunocompromised) may be more likely to get vaccinated than the general public. In this case, the realized value of $\theta_{\lambda} = \frac{\lambda_{FV}}{\lambda_{FU}}$ is closer to 1 than it would be if it only captured immunological phenomena.
```